package.json

```diff
@@ -14,11 +14,13 @@
     "build:frontend": "yarn workspace @asap-hub/frontend build",
     "build:frontend:typecheck": "yarn workspace @asap-hub/frontend typecheck",
     "build": "yarn build:typecheck && yarn build:babel && yarn build:frontend:typecheck && yarn build:frontend",
+    "watch:babel": "yarn workspaces foreach -pi -j 1000 run watch:babel",
+    "watch:typecheck": "yarn build:typecheck -w",
+    "dev": "cross-env NODE_ENV=development yarn sls offline",
+    "sls": "node serverless/link-plugins.js && yarn pnpify serverless",
```

Comment:
This is awesome I think, now CI and `dev` care neither about `serverless` being `pnpify`ed nor about the initialization step, they just let `yarn sls` handle it :ok_hand: